test/FsAutoComplete.IntegrationTests/MultipleUnsavedFiles/output.json

```diff
@@ -73,7 +73,7 @@
         "StartColumn": 23,
         "EndColumn": 29,
         "Severity": "Error",
-        "Message": "The value, constructor, namespace or type 'addTwo' is not defined.",
+        "Message": "The value, constructor, namespace or type 'addTwo' is not defined. Maybe you want one of the following:\n   addTwo2",
```

Comment:
Thank you @forki !
I was really sad to change that last time to not include the suggestion :(

Is is really awesome feature